File: annotation/src/main/java/com/cangwang/annotation/ModuleBean.java
package com.cangwang.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Module
 * Created by cangwang on
 */
@Target({ElementType.TYPE})
@Retention(RetentionPolicy.CLASS)
public @interface ModuleBean {
}

File: kotlinannotation/src/main/java/com/cangwang/annotation/ModuleBean.kt
package com.cangwang.annotation

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Module
 * Created by cangwang on
 */
@Target(AnnotationTarget.ANNOTATION_CLASS, AnnotationTarget.CLASS)
@Retention(RetentionPolicy.CLASS)
annotation class ModuleBean 